### PR TITLE
Add missing atmel-samd and nrf52840 bootloader IDs

### LINKED ISSUE #943: Bootloader section missing for some boards
For example, not showing up for Matrix Portal:

image

### PR DESCRIPTION
**Pull request by makermelissa**
Fixes #943.

### CODE REVIEW

File: _board/ndgarage_ndbit6_v2.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ board_image: "unknown.jpg"
 downloads_display: false
 date_added: 2020-09-1
 family: atmel-samd
+bootloader_id: ndbit7
```
Comment (kattni) [inline _board/ndgarage_ndbit6_v2.md:12]: Is this supposed to be `ncbit6`?

**Review by kattni — CHANGES_REQUESTED** [review]
Looks good! Caught one thing below.

File: _board/ndgarage_ndbit6_v2.md (hunk 1 above)
Comment (makermelissa) [reply _board/ndgarage_ndbit6_v2.md:12]: I don't believe so. There was already an ndbit6 board (different than this) and there is nothing called ndbit7, so I figured they were being clever.

Comment (kattni) [reply _board/ndgarage_ndbit6_v2.md:12]: Fair enough!

**Review by kattni — APPROVED** [review]

**Pull request merged**